tests/Fixtures/filters/round.test

```diff
@@ -1,6 +1,7 @@
 --TEST--
 "round" filter
 --TEMPLATE--
+{{ null|round }}
```

Comment:
Can you also add two other tests for the `ceil` and `floor`  methods?

Reply:
`floor` is already tested (that's the last case), but `null`  is indeed missing.

Reply:
Done